test/convergence/test_mini_models.py

```diff
@@ -253,6 +258,50 @@
             attn_implementation="sdpa",  # default value, pytorch native attention
         ),
     ),
+    "mini_qwen2_vl": MiniModelConfig(
```

Comment:
These tests test the llm portion of qwen2_vl, but not the vit portion (which LayerNorm affects)